tutorials/32_Language_Classification_In_Documents_And_Queries.ipynb

```diff
@@ -0,0 +1,1501 @@
+{
```

Comment:
<ol><li>Let's add the links to all aailable documentation pages</li><li>After 'Goal' can we again say 'Optionally, at the end you'll also see how to incorporate language detection...'</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>